doc/release/release_2.6.rst

```diff
@@ -279,7 +279,7 @@ Merged PRs
 - Speed up transitivity, remove redundant call (#4198)
 - NXEP 2 — API design of view slices (#4101)
 - Cleanup old platforms (#4202)
-- Fixed "topolgical_sort" typo (#4211)
+- Fixed "topological_sort" typo (#4211)
```

Comment:
I try not to change typos in changelogs.

In general, I'd encourage people not to include typos in changelogs, it's confusing.

Happy to drop this.